returnn/torch/engine.py

```diff
@@ -530,6 +537,9 @@ def eval_model(self, *, skip_already_evaluated: bool = False):
 
             with torch.no_grad():
                 for extern_data_raw in data_loader:
+                    if self._torch_distributed_ctx and step_idx % 100 == 0:
+                        torch.distributed.broadcast(_has_data, src=0)
```

Comment:
I think it's more readable if code is more local.
```suggestion
                        _has_data = torch.Tensor([True]).to(device="cpu", dtype=torch.int8)
                        torch.distributed.broadcast(_has_data, src=0)
```